CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 Currently, this repo is in Prerelease. When it is released, this project will adhere to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
 ========
 
+## 0.18.4
```

Comment:
@scooooooooby , the `Tabs` component is still being worked on by UX so maybe we shouldn't release this WIP? Maybe it doesn't really matter since DS isn't `1.x.x` yet.
Or maybe the code can be merged but not exported until it's fully ready.

Reply:
That makes sense to me. Let's hold off and get it all correct in Main first.

Reply:
To be clear, does this mean I'm indefinitely holding off on doing more work on this, or...?

Reply:
@adriana-alter For the time being, yes. I'll work on getting #410 properly triaged for you today.

Reply:
Okay, thanks :)

Reply:
Might actually happen the week after Thanksgiving now that I'm in it—I'll need Kathy to approve the designs first, and I don't know what her day is like. For now you're off the hook though Adriana, thanks for getting us started here!

Reply:
Happy Thanksgiving, and looking forward to CSS-ing (or whatever else you end up needing)! 😃